waku/v2/protocol/waku_message.nim

```diff
@@ -31,6 +31,14 @@ type
     # this field will be used in the rln-relay protocol
     # XXX Experimental, this is part of https://rfc.vac.dev/spec/17/ spec and not yet part of WakuMessage spec
     proof*: RateLimitProof
+    # The ephemeral field indicates if the message should
+    # be stored. bools and uints are 
+    # equivalent in serialization of the protobuf
+    ephemeral*: bool
+    # The storeTTL field indicates for how long a given message
+    # should be considered "valid", after which it may still
+    # be fetched, but should not be used in any business logic
+    storeTTL*: Timestamp
```

Comment:
What does this timestamp represent?

 * The point in time after which the message will be no longer valid? This is what I think when I see the "timestamp" type.
 * The amount of time that the message will be valid. Then which is the starting point in time? Note that the _senderTimestamp_ field in the message is optional.

Reply:
If the senderTimestamp is optional, then storeTTL can be used to indicate the timestamp after which the message is invalid. wdyt?

Reply:
Agree that this should be part of a design discussion centred around RFC changes in https://github.com/vacp2p/rfc. Have discussed this out of band and will continue the conversation there once the proposed changes are ready for review.